app/frontend/src/components/forms/RequestReceipt.vue

```diff
@@ -20,29 +20,49 @@ const properties = defineProps({
     type: String,
     required: true,
   },
+  formId: {
+    type: String,
+    required: true,
+  },
 });
 
 const emailRules = ref([(v) => !!v || 'E-mail is required']);
-const form = ref(null);
+const forms = ref(null);
 const priority = ref('normal');
 const showDialog = ref(false);
 const to = ref('');
+const formStore = useFormStore();
 
-const { isRTL } = storeToRefs(useFormStore());
+const { isRTL, form } = storeToRefs(useFormStore());
 
 onMounted(() => {
   resetDialog();
+  formStore.fetchSubmission({ submissionId: properties.submissionId });
 });
 
 function displayDialog() {
   showDialog.value = true;
 }
 
 async function requestReceipt() {
-  const { valid } = await form.value.validate();
+  const { valid } = await forms.value.validate();
   if (valid) {
     const notificationStore = useNotificationStore();
     try {
+      if (form.value.enableTeamMemberDraftShare) {
+        const formUsersResponse = await rbacService.isUserAssignedToFormTeams({
+          formId: properties.formId,
+          email: to.value,
+          roles: '*',
+        });
+        if (formUsersResponse && !formUsersResponse.data) {
+          notificationStore.addNotification({
+            ...NotificationTypes.ERROR,
+            text: `You can't share the draft with users who haven't been added to the form. Please contact the admin to add them.`,
```

Comment:
We need the internationalization for all the languages. We expended some effort to get all the languages up to date and don't want to add any tech debt, so we keep it up to date in each PR.
"Draft submissions may only be shared with team members. Contact the form owner to add them."